content/blog/beta-test-our-new-hpe-greenlake-for-data-fabric.md

```diff
@@ -0,0 +1,34 @@
+
+Today we have returned to the single data lake as one solution to the many versions of the truth issue. The single date lake is reminiscent of past approaches: bring all data into one location to standardize, normalize, aggregate and then distribute the results. A second approach, a federated approach, is to leave the data in place and retrieve only what is requested when it is requested, possibly with some query optimization and intelligent caching thrown in to minimize the impact on performance.
+
+**Solving the issue with data fabric**
+
+HPE Ezmeral Data Fabric started and grew up in the big data world. It has long been trusted as an enterprise-grade data fabric managing data at scale for mission-critical applications across industries and use cases. Users today enjoy a single global name space that provides one point to review all attached data assets across data fabric deployments. Multi-modal data access delivers on the need to build an application once and deploy anywhere there is a data fabric. In addition, the multi-modal nature of the data fabric allows the development of stream-based, object-based apps and file-based apps against the same data without duplicating that data – very much a single version of the truth. Additionally, and if needed, the data fabric is good at moving data securely and optimally from deployment to deployment. 
+
+The upcoming release of HPE GreenLake for Data Fabric brings the same enterprise-grade data fabric to the as-a-Service world. The installation, configuration and management of deployments is no longer a customer responsibility. The data fabric comes installed, running and is maintained by HPE. Customer-facing management tasks are elevated to the level of creation of storage buckets and volumes, managing user quotas, sharing of entry points with fellow application developers and overall monitoring of usage, all driven through a central UI that can manage all your HPE GreenLake for Data Fabric deployments.
+
+**Beta test it yourself**
+
+With the upcoming HPE GreenLake for Data Fabric beta, we are targeting use cases in the hybrid and multi-cloud world, for example those who want to write an application once and run it on any deployment target using a consistent set of multi-modal APIs. It is for those who want to build a data plane that gives a consistent view of data assets across clouds via a single global name space and those that want to synchronize data across clouds securely and efficiently. The as-a-Service tooling allows the management of all clusters from a single interface, further simplifying management of your data plane. We will post additional resources on developer.hpe.com that show example use cases.
+
+HPE will begin a beta program of this new service in November this year (2022). Beta testers, can deploy initially into AWS then other public clouds as well as on-prem hardware. Join us to learn more in our upcoming webinar. We look forward to learning about your use cases.
```

Comment:
Join us to learn more in our upcoming webinar....

I assume that a registration link will be added soon. Correct?

Reply:
Got it. Thank you for your review!